zip-archive-0.3.2 is failing tests on nightly-2018-01-17

zip-archive-0.3.2 is failing tests on nightly-2018-01-17 with the following output:

```
[1 of 1] Compiling Main             ( tests/test-zip-archive.hs, dist/build/test-zip-archive/test-zip-archive-tmp/Main.o )

tests/test-zip-archive.hs:12:1: warning: [-Wunused-imports]
    The import of ‘Control.Applicative’ is redundant
      except perhaps to import instances from ‘Control.Applicative’
    To import instances alone, use: import Control.Applicative()
   |
12 | import Control.Applicative
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
Linking dist/build/test-zip-archive/test-zip-archive ...
> /tmp/stackage-build12/zip-archive-0.3.2$ dist/build/test-zip-archive/test-zip-archive
^MCases: 8  Tried: 0  Errors: 0  Failures: 0^MCases: 8  Tried: 1  Errors: 0  Failures: 0^MCases: 8  Tried: 2  Errors: 0  Failures: 0^M                                          ^M### Error in:   2
tests/test_dir_with_symlinks: getSymbolicLinkStatus: does not exist (No such file or directory)
^MCases: 8  Tried: 3  Errors: 1  Failures: 0^MCases: 8  Tried: 4  Errors: 1  Failures: 0  adding: LICENSE (deflated 46%)
  adding: src/ (stored 0%)
  adding: LICENSE (deflated 46%)
  adding: src/ (stored 0%)
  adding: src/Codec/ (stored 0%)
  adding: src/Codec/Archive/ (stored 0%)
  adding: src/Codec/Archive/Zip.hs (deflated 74%)
^M                                          ^M### Error in:   4
tests/test_dir_with_symlinks: getSymbolicLinkStatus: does not exist (No such file or directory)
^MCases: 8  Tried: 5  Errors: 2  Failures: 0^MCases: 8  Tried: 6  Errors: 2  Failures: 0  creating: test-zip-archive.1078/dir1
extracting: test-zip-archive.1078/dir1/hi
  creating: test-zip-archive.1078/dir1/dir2
 inflating: test-zip-archive.1078/dir1/dir2/hello
^MCases: 8  Tried: 7  Errors: 2  Failures: 0  creating: test-zip-archive.1078/dir3
extracting: test-zip-archive.1078/dir3/hi
^M                                          ^MCases: 8  Tried: 8  Errors: 2  Failures: 0
```
We have added it to the `expected-test-failures` list &mdash; please let us know when you think the tests should pass.

--- a/stack.yaml
+++ b/stack.yaml
@@ -2,7 +2,6 @@ resolver: lts-10.3
 packages:
 - '.'
 extra-deps:
-- directory-1.3.1.5
 flags:
   zip-archive:
     splitbase: true

--- a/tests/test-zip-archive.hs
+++ b/tests/test-zip-archive.hs
@@ -38,7 +38,7 @@ createTestDirectoryWithSymlinks prefixDir  baseDir = do
   createDirectoryIfMissing True (testDir </> "1")
   writeFile (testDir </> "1/file.txt") "hello"
   cwd <- getCurrentDirectory
-  createFileLink (cwd </> testDir </> "1/file.txt") (testDir </> "link_to_file")
+  createSymbolicLink (cwd </> testDir </> "1/file.txt") (testDir </> "link_to_file")
   createSymbolicLink (cwd </> testDir </> "1") (testDir </> "link_to_directory")
   return testDir
 

--- a/zip-archive.cabal
+++ b/zip-archive.cabal
@@ -66,7 +66,7 @@ Test-Suite test-zip-archive
   Main-Is:        test-zip-archive.hs
   Hs-Source-Dirs: tests
   Build-Depends:  base >= 4.2 && < 5,
-                  directory >= 1.3.1, bytestring >= 0.9.0, process, time, old-time, 
+                  directory >= 1.3, bytestring >= 0.9.0, process, time, old-time, 
                   HUnit, zip-archive, temporary, filepath
   Default-Language:  Haskell98
   Ghc-Options:    -Wall

